src/page_body/structure/blocks/page_block_figma_component_props_table.pr

```diff
@@ -0,0 +1,97 @@
+{[ const block = context /]}
+
+{[ let configuration = exportConfiguration() /]}
+
+{[ const brands = ds.allBrands() /]}
+{[ const components = ds.allDesignComponents(brands[0].id) /]}
+
+<div>
+    {[ for component in components ]}
+        {[ if (component.id === block.selectedFigmaComponent.figmaComponentId) ]}
+            {[ if block.figmaComponentsBlockConfig.showComponentName ]}
+                <div class="props-table-component-name">{{ component.name }}</div>
+            {[/]}
+            <table class="props-table">
+                <thead>
+                    <tr>
+                    <th>
+                        Property
+                    </th>
+                    <th>
+                        Values
+                    </th>
+                    <th>
+                        Default
+                    </th>
```

Comment:
We should probably fixed formatting and indentation to be more readable

Reply:
[0dc555f](https://github.com/Supernova-Studio/exporter-documentation/pull/166/commits/0dc555f0e644016d8d8a3c05df3153c0c0139324)